Gopkg.toml

```diff
@@ -41,20 +41,20 @@ required = [
   version = "kubernetes-1.13.1"
 
 [[constraint]]
-  name="sigs.k8s.io/controller-tools"
-  version="v0.1.1"
+  name = "sigs.k8s.io/controller-tools"
+  version = "v0.1.9"
 
 [[constraint]]
-  name="github.com/softlayer/softlayer-go"
-  branch="master"
+  name = "github.com/softlayer/softlayer-go"
+  revision = "7c592eb"
 
 [[constraint]]
   name = "k8s.io/klog"
-  version = "0.3.0"
+  version = "0.3.2"
 
 [[constraint]]
   name = "sigs.k8s.io/cluster-api"
-  branch = "release-0.1"
+  revision = "280e527"
```

Comment:
what is the reason to change it? As my comments above, we'd better use stable release for such kind of project.
@jichenjc @clyang82

Reply:
In order to leverage new feature of `cluster-api`, for example: generate random cluster name instead of hardcode `clusterapi`. that will enable to run the job in parallel.

Reply:
@xunpan https://github.com/kubernetes-sigs/cluster-api/pull/984

Reply:
@xunpan that's my bad, I thought this is for softlayer ... I also think we need stick to release branch 
per above comments, if we need some feature, we can backport from master to release branch

@clyang82 can you help to revert this line back ? THanks

Reply:
@xunpan @jichenjc Consider master is a development branch, we can update the `cluster-api` to a new stable version after the release including kubernetes-sigs/cluster-api#984 was cut. For now, how about keep the current code as it is? @clyang82 can follow up another PR to upgrade the cluster-api later.

Reply:
@clyang82 I'm not sure.
The new code always fix many bugs. No doubt about it. The thing is that we should balance the code maintenance and bug fix for 3rd parties. I think only block issue can break the rule for maintenance. Or else, waiting for new stable release should be better.

Reply:
+1 to @xunpan , I don't think switch to master develop branch is a very good idea for us at current stage

Reply:
agree with your idea. Here is new PR #239 to revert the change for clusterapi branch. Thanks.